src/theme/theme-wrapper.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+
+import { deepMerge } from '../helpers';
+
+import { darkTheme, lightTheme } from './themes';
+import { ThemeProvider, useTheme } from './theme-context';
+
+import type {
+  WafflesTheme,
+  WafflesThemeConfig,
+  WafflesThemeOverride,
+} from './themes/types';
+
+const WAFFLES_THEME_MAP = {
+  light: lightTheme,
+  dark: darkTheme,
+} as const;
```

Comment:
```suggestion
} as { [theme in WafflesTheme]: typeof lightTheme };
```

We’d like to get warned somewhere (doesn’t have to be here) against mismatches (missing key/value pairs) between `lightTheme`, `darkTheme` and/or any other future themes